### ISSUE TITLE
Add Pluggins impossible

### ISSUE THREAD

**Issue by Jhobean** [polserver/UOFiddler#18]
When I open the pluggins screen I always have this message:
![image](https://user-images.githubusercontent.com/51728381/78250096-5ad87e80-74bd-11ea-83b8-772dc2e3ad86.png)

I restart 20 times and its always the same

**Comment by AsYlum-** [comment]
Which version of UOFiddler are you using?
In your UOFiddler folder do you have "plugins" folder with *.dll files?

**Comment by AsYlum-** [comment]
Ok. I've deleted duplicated comment. In next few days I will try to add some logging to plugin loading code and we will go from there. I will post update here when it's done. 

In meantime you can try one more thing. Go to  %APPDATA% in your file explorer and rename UOFiddler folder to for example UOFiddler.old and start application. It will run with default configuration. In this case is the manage plugins window still empty?

**Comment by Jhobean** [comment]
Cannot boot fiddler anymore if i Rename my \AppData\Roaming\UoFiddler.old
![image](https://user-images.githubusercontent.com/51728381/78309139-3493fc80-7518-11ea-9afc-20d4bea387c7.png)

**Comment by AsYlum-** [comment]
Welp... looks like there is also another issue when copying configuration files :) 

So first of all get latest pre-release from here https://github.com/polserver/UOFiddler/releases/tag/4.9.1 then please download and unpack it to some folder but don't run it yet. Rename or remove that AppData\Roaming\UOFiddler folder and now run the UOFiddler.

Also can you message me on discord? My username is: AsYlum-#3853 and here is invitation for polserver discord server: https://discord.gg/42XJxq (valid 24hours).

**Comment by Jhobean** [comment]
Made some test. With the new 4.9.1 version 
- renaming in appdata .old. I can lauch fiddler and a new file is create
- on my laptop work well
- On my PC same message Restart needed
- Here the position of my fiddler folder (C:\UOFiddler4.9.1\UoFiddler.exe)
Both PC has same Config (french canadian)

**Comment by AsYlum-** [comment]
Ok the bug is now almost solved. But if you encounter this there is workaround. Go to the UOFiddler folder and check all file properties (right click plugin file -> Properties)

![unblock_net_file](https://user-images.githubusercontent.com/377468/78463796-7623d380-76e1-11ea-9320-6d70a0b6ede4.png)

If the file has unblock options you need to check it. You need to do this for all files sadly :(

**Comment by Jhobean** [comment]
Thx working:smile:
